src/styles/utilites/_variables.scss

```diff
@@ -0,0 +1,8 @@
+$white-color: #fff;
+$aqua-color: #00acdc;
+$dark-blue-color: #060b35;
+$light-grey-color: #616070;
+$creame-color: #f3f3f3;
+
+$card-width: 200px;
+$distance-between: 48px;
```

Comment:
Put more clearly name for varible `$distance-between: 48px;` to make varible more understandable